sysutils/cfm/Portfile

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        willeccles cfm 0.5.5 v
+categories          sysutils
+platforms           darwin
+license             MPL-2
+maintainers         {eccles.dev:will @willeccles}
+description         Cactus File Manager
+long_description    Cactus File Manager is a simple and fast TUI file manager
+homepage            https://github.com/willeccles/cfm
+
+checksums           rmd160  e864a6d96f170f5013df61496440bfd0597a7950 \
+                    sha256  22c52896986883e0d831b75d7b1c01217cac04043357c3fab2527eb363d0600d \
+                    size    70980
+
+destroot.args       DESTDIR=${destdir} PREFIX=${prefix}
```

Comment:
I assume you are talking about the DESTDIR and PREFIX. I added those because without that line, it will not install properly, as cfm's Makefile doesn't read the `destdir` and `prefix` variables. It expects the capitalized versions. After adding this line, it will install properly.